_data/projects/iyan-3d.yml

```diff
@@ -0,0 +1,17 @@
+name: Iyan 3D
+desc: 3D animation app for iOS & Android
+site: https://github.com/codetiger/Iyan3d
+tags:
+- 3d
+- animation
+- opengles
+- ios-metal
+- opengl
+- ios
+- android
+- raytracing
+- rendering
+- physics
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/codetiger/Iyan3d/labels/up%20for%20grabs
```

Comment:
@codetiger I noticed that this repo doesn't have any tasks related to this label as of now. Would you please add some tasks with this label?

Reply:
Hi @ritwik12,
I've tagged few tags in the label name "up for grabs" in the repo to start with. In the coming few days I'll be adding a roadmap as well. Thanks for the feedback.